.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  'env': {
+    'commonjs': true,
```

Comment:
Looks like this is redundant with `'node': true`

Reply:
This file was autogenerated (I picked Google's JS standard, which I assume we theoretically adhere to), I was inclined not to mess with it but I can remove this if you want.